### PR TITLE
219 bundle documentation

### LINKED ISSUE #219: Bundle documentation
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### Please describe the feature you have in mind and explain what the current shortcomings are?

We need documentation explaining the bundle concept to help users understand these basic concepts.

### How would you imagine the implementation of the feature?

- What is a bundle
- How addon settings work
- production, staging, development
- working with bundles
- updating addons
- uninstalling addons

### Are there any labels you wish to add?

- [X] I have added the relevant labels to the enhancement request.

### Describe alternatives you've considered:

_No response_

### Additional context:

_No response_

### LINKED ISSUE #226: Copy Settings Documentation
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### Please describe the feature you have in mind and explain what the current shortcomings are?

There are a lot of different Settings in AYON.

- Anatomy
- Studio Settings
- Project Settings

And there are a lot of ways to "copy paste" these types of settings, and potentially more coming with https://github.com/ynput/ayon-frontend/pull/752 and reported [here](https://github.com/ynput/ayon-frontend/pull/752#issuecomment-2312283193) we may need to document the different types of "Settings Management" and how to copy e.g. specific settings that you'd want from one place to another.

But a user may also face issues with **Copy** and **Paste** in the settings pages as this issue describes: https://github.com/ynput/ayon-frontend/issues/762 
So we may want to also highlight that and maybe [have some graphics displaying how to enable that](https://github.com/ynput/ayon-frontend/issues/762#issuecomment-2315178427) using `tip` section in the documentation.

### How would you imagine the implementation of the feature?

We may need some very clear wording in the documentation that we have a few levels of granularity to copying settings:

1. Copy settings from... on **bundles page** copies all studio settings and project overrides for that bundle* (not released yet)
2. Copy settings from... on **studio settings** page copies only the studio settings between the bundles
3. Copy settings from... on **studio settings page on a particular addon** copies between studio settings for only a specific addon version.
4. Copy settings from... on **project settings** page copies only the project settings for the selected project between the bundles.
5. Copy settings from... on **project settings page on a particular addon** copies project overrides for only selected project for only specific addon version.

The ["Copy Settings" toggle on creating a new bundle is basically that top one (number 1) described above](https://github.com/ynput/ayon-frontend/pull/752) - it will copy all studio settings and project setting overrides between the bundles.

That's a lot of different dialogs we have. And someone may be as stupid as me and definitely at one point will by accident be using the wrong page, using the wrong dialog unintentionally.

Also #219 will be crucial because to understand settings we'd also need to document that

- Addons have settings PER variant PER addon version.

> Because that is the hardest concept to grasp that you're copying to variant for each addon version... and not to the bundle.

### Are there any labels you wish to add?

- [X] I have added the relevant labels to the enhancement request.

### Describe alternatives you've considered:

Better settings UIs and dialogs will definitely help already.

### Additional context:

The current settings documentation is old anyway, with screenshots that come mostly from OpenPype's Qt dialogs still: https://ayon.ynput.io/docs/admin_settings

So some love for this page is more than welcome!

[Issues with Copy & Paste also came up on the forum](https://community.ynput.io/t/paste-anatomy-results-in-clipboard-is-empty/1274/6)

### PR DESCRIPTION
**Pull request by MustafaJafar**
## Changelog Description
resolve #219 
resolve #226 

Changes: 
- [x] Explaining Bundles and addons
- [x] Installing Addons and Creating Bundles
- [x] Adding Information about dependency packages
- [x] Pipeline modes and settings
- [x] Different ways to copy settings
- [x] Show Settings filter and how to use it.

## Additional Info
Some related discussions in forums that add food for thought.
- [Copying settings between bundles](https://community.ynput.io/t/copying-settings-between-bundles/1867)
- [Copy addon settings between bundles | announcement](https://community.ynput.io/t/ayon-server-1-3-9-settings-migration-tool/1734#copy-addon-settings-between-bundles-2)

## Testing notes:
1. Read through changes and approve if they make sense to you.

### CODE REVIEW

File: website/docs/admin_server_bundles_and_addons.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+Dev variant is the same as dev bundle name `<dev-bundle-name>`
+Dev bundle are only accessible for the **assigned admin user**.
+:::
+
+### How Settings work
+
+This variant is used across the addons.  
+Each addon is saved on the server with different variant of settings.
+The variant used in addons is exposed on the bundle.
+The variant value on the bundle is like a main switch that set all the addons to use that variant value.
```
Comment (BigRoy) [inline website/docs/admin_server_bundles_and_addons.md:36]: I couldn't make any sense of this myself. 🗡️ 

Which variant? How is it used? Etc. It tells me too little about settings and more about variants and bundles.

Maybe the "title" of the section is what confuses me "How settings work" - but it doesn't seem to be about that at all?

I feel like this community forum topic has some great wording explaining how settings actually work and behave: https://community.ynput.io/t/copying-settings-between-bundles/1867

File: website/docs/admin_server_bundles_and_addons.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+:::tip Settings management
+For more info about setting management go to [Addon Settings](admin_server_settings_management.md#addon-settings)
+:::
+
+### Working with bundles
+#### Create bundles
+You can create a bundle by either one of these ways:
```
Comment (BigRoy) [inline website/docs/admin_server_bundles_and_addons.md:44]:
```suggestion
You can create a bundle in multiple ways: 
```

File: website/docs/admin_server_bundles_and_addons.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+#### Configure bundles
+
+Bundles are configured on creation where you specify
+- Bundle name
+- Launcher version
+- Dev bundle toggle
+- Addon versions
+
+After Creation these are locked and you can then only change
+- Dependency Packages.
```
Comment (BigRoy) [inline website/docs/admin_server_bundles_and_addons.md:58]: You can also still specify for which variant this bundle is active, like production, dev, etc.

File: website/docs/admin_server_bundles_and_addons.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+- Change version of some server addons (that don't affect the pipeline). e.g. Jira Addon.
+
+:::note
+Dev bundle supports more configurations: 
+- Assigned dev
+- Addon versions
+- Addon client code directory. This special one allows developers to test the client code directory without uploading the addon. 
+  However, they'll still need to upload the addons to update the settings on the server.
```
Comment (BigRoy) [inline website/docs/admin_server_bundles_and_addons.md:66]: If we have it - this is a perfect place to just link to developer mode documentation?

File: website/docs/admin_server_bundles_and_addons.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+:::
+
+### Working with Addons
+#### Install Addons
+
+- `Addon Market`
+- `Upload Addons`
+
+:::tip
+When updating pipeline release, new addons will be installed.
```
Comment (BigRoy) [inline website/docs/admin_server_bundles_and_addons.md:76]: Similar to what the pop-up mention states it may be worth it to describe here that the addons are added to a new inactive bundle for you to test. It doesn't update your live running bundles or productions.

> This will not affect your existing setup, a new bundle will be created for testing first.

image

File: website/docs/admin_server_bundles_and_addons.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+:::
+
+#### Uninstall Addons
+- `Uninstall Versions` button.
+
+:::caution
+WIth local server deployment,
```
Comment (BigRoy) [inline website/docs/admin_server_bundles_and_addons.md:83]:
```suggestion
With local server deployment, 
```

File: website/docs/admin_server_bundles_and_addons.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+When updating pipeline release, new addons will be installed.
+:::
+
+#### Uninstall Addons
+- `Uninstall Versions` button.
+
+:::caution
+WIth local server deployment, 
+You can add or remove addons by extracting and deleting files from your addons directory on your ayon-docker server.
+:::
```
Comment (BigRoy) [inline website/docs/admin_server_bundles_and_addons.md:85]: I'm worried this may have unforeseen consequences and isn't something we should recommend in any way. 
This should really be a last resort.

@martastain thoughts?


If we want to keep this I'd rephrase it:

> Technically it is possible to add or remove addons to the server by extracting to or deleting files from your addons directory on the `ayon-docker` server directly. However, we recommend using the AYON frontend or API to manage the addons to ensure the correct behavior. Managing the files on the server directly should be a last resort.

File: website/docs/admin_server_bundles_and_addons.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+
+:::info
+Developers can upload/remove addons via ayon python api.
```
Comment (BigRoy) [inline website/docs/admin_server_bundles_and_addons.md:88]: This would be best with an example, or a link to documentation that shows that particular part of that API.

File: website/docs/admin_server_settings_management.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+- Anatomy Presets
+- Anatomy Settings
+- Roots Override
+
+## Addon Settings
+
+*Addon settings are accessed via Bundles.*
+
+- Studio Settings: Addon settings on studio level.
+- Project Settings: Addon settings on project level. They inherit studio settings and allowing overrides per project.
```
Comment (BigRoy) [inline website/docs/admin_server_settings_management.md:27]:
```suggestion
- Project Settings: Addon settings on project level. They inherit studio settings and allow pinned overrides per project. A pinned override does not inherit from studio settings anymore and is explicitly set on the project settings.
```

File: website/docs/admin_server_settings_management.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+- Studio Site Settings: Addon local settings on studio level. They don't inherit (nor override) any other settings.
+- Project Site Settings: Addon local settings on project level. They don't inherit (nor override) any other settings.
```
Comment (BigRoy) [inline website/docs/admin_server_settings_management.md:29]: Should this state that it's _per site id_? Or what is "local" in this case?

**Review by BigRoy — CHANGES_REQUESTED** [review]

File: website/docs/admin_server_bundles_and_addons.md [hunk 11]
```diff
@@ -5,30 +5,30 @@ sidebar_label: Bundles & Addons
...
-more info check [How to use different bundles with different projects?](https://community.ynput.io/t/how-to-use-different-bundles-with-different-projects/1096)
+You can override the default behavior and explicitly specify a bundle on launch, see:
+[How to use different bundles with different projects?](https://community.ynput.io/t/how-to-use-different-bundles-with-different-projects/1096)
 :::
         
-:::note
-Dev variant is the same as dev bundle name `<dev-bundle-name>`
-Dev bundle are only accessible for the **assigned admin user**.
+:::note Dev variants
+For a dev bundle the its dev variant has the sae name `<dev-bundle-name>`.
```
Comment (NeirdaP) [inline website/docs/admin_server_bundles_and_addons.md:26]: Two small typos :)

```suggestion
For a dev bundle its dev variant has the same name `<dev-bundle-name>`.
```

**Comment by MustafaJafar** [comment]
This PR has received new changes that follows completely a different way to explain the topic.

e.g. when explaining addons, I reused features cards from [features](https://ayon.ynput.io/features) page to explain the addons concept instead of providing a detailed description about addons.

image

File: website/docs/admin_server_bundles_and_addons.md [hunk 12]
```diff
@@ -0,0 +1,289 @@
...
+![Settings and Overrides Color Code](assets/server/admin/server_and_addons/settings_and_overrides_colorcode.png)
+
+Settings and their overrides use the following color code:
+
+- **No Color**: Default value
+- <span style={{color:'#25d15f'}}>▌</span>: Studio override
+- <span style={{color:'#ff9e3d'}}>▌</span>: Project override
+
+### Settings Filter
+
```
Comment (MustafaJafar) [inline website/docs/admin_server_bundles_and_addons.md:226]: After an internal discussion on discord, let's mention when the feature was introduced. 
> "Introduced in 1.5.4" as a footnote.

Comment (MustafaJafar) [reply website/docs/admin_server_bundles_and_addons.md:229]: https://github.com/ynput/ayon-documentation/pull/298/commits/3bf76706cc2c1a5bb594d9dace3a74d785d84fc1

File: website/docs/admin_server_bundles_and_addons.md [hunk 13]
```diff
@@ -0,0 +1,292 @@
...
+   - **+ Add Bundle**: Create a new bundle from scratch.
+   - **Upload Addons**: Upload addon `.zip` packages to your server.
+   - **Upload Launcher**: Upload an AYON Launcher.
+   - **Upload Dependency Package**: Upload a dependency package.
+   - **Show Archived**: Enable to display archived bundles.
+   - **Set Staging**: Assign a `Staging` status to your bundle.
+   - **Set Production**: Assign a `Production` status to your bundle.
+   - **Duplicate and Edit**: Create a new bundle from the selected bundle.
+
+:::info Upload AYON Products
```
Comment (BigRoy) [inline website/docs/admin_server_bundles_and_addons.md:102]: I think "Products" is the wrong terminology here. It should be `Addons`, right?

Products conflicts to much with the actual published products too.

EDIT: I suppose you're referring to "Products" due to it being either the addons, launcher or dependency packages and not just addons. It just seems quite confusing compared to the production terminology.

File: website/docs/admin_server_bundles_and_addons.md [hunk 14]
```diff
@@ -0,0 +1,292 @@
...
+
+Most of the time, you can make various AYON products available on your server using the [Addon Market](admin_server_market.md) and by [installing pipeline releases](admin_server_updating_pipeline.md).
```
Comment (BigRoy) [inline website/docs/admin_server_bundles_and_addons.md:104]: Would this be better?
```suggestion
You can easily install new or update existing AYON addons on your server using the [Addon Market](admin_server_market.md) and by [installing pipeline releases](admin_server_updating_pipeline.md).
```

File: website/docs/admin_server_bundles_and_addons.md [hunk 15]
```diff
@@ -0,0 +1,292 @@
...
+
+However, when customizing your pipeline, you'll often need to upload different AYON products manually. Use the **Upload Addons**, **Upload Launcher**, and **Upload Dependency Package** buttons for this process.
```
Comment (BigRoy) [inline website/docs/admin_server_bundles_and_addons.md:106]: Should there be a hint somewhere that Ynput cloud hosted installs that are not on an active paid subscription may have locked uploading of custom addon `.zip` to avoid potential abuse of the Ynput cloud servers ? @mkolar

File: website/docs/admin_server_bundles_and_addons.md [hunk 16]
```diff
@@ -0,0 +1,292 @@
...
+
+1. **Bundle Name**: The name of the bundle.
+2. **Launcher Version**: The assigned version of the AYON launcher.
+3. **Dev Bundle**: Enable this to set the bundle to `Dev`. *This setting is available when [Developer Mode](dev_dev_mode.md) is enabled.*
+4. **Assigned Dev**: The developer assigned to the dev bundle. *This setting is available when [Developer Mode](dev_dev_mode.md) is enabled.*
+5. **Addons List**: A list of addons included in your bundle, where you can specify the version of each addon.
+6. **Compatibility Checker**: Ensures your selected addons meet compatibility requirements with other AYON products in the bundle. Learn more in [Understanding Dependencies in AYON Addons | Ynput Forums](https://community.ynput.io/t/understanding-dependencies-in-ayon-addons/1992).
+
+:::info
+After creation, the previously mentioned options will be locked. You can only change:
```
Comment (BigRoy) [inline website/docs/admin_server_bundles_and_addons.md:143]:
```suggestion
After a bundle is created, most of the options will be locked. You can only change:
```

File: website/docs/admin_server_bundles_and_addons.md (hunk 13 above)
Comment (MustafaJafar) [reply website/docs/admin_server_bundles_and_addons.md:102]: you are right, what could be a suitable term to refer to AYON artifacts either the addons, launcher or dependency packages and not just addons.

File: website/docs/admin_server_bundles_and_addons.md (hunk 14 above)
Comment (MustafaJafar) [reply website/docs/admin_server_bundles_and_addons.md:104]: same comment as https://github.com/ynput/ayon-documentation/pull/298#discussion_r1909458904
as I'd like refer to the different types to AYON artifacts.

File: website/docs/admin_server_bundles_and_addons.md (hunk 13 above)
Comment (BigRoy) [reply website/docs/admin_server_bundles_and_addons.md:102]: Tagging @mkolar @Innders - they may have an idea on terminology here.

File: website/docs/admin_server_bundles_and_addons.md [hunk 17]
```diff
@@ -0,0 +1,292 @@
...
+
+:::tip Override Pipeline Mode
+The AYON launcher lets you override the default behavior, allowing you to specify a bundle and pipeline mode. For more details, visit [How to use different bundles with different projects? | Ynput Forums](https://community.ynput.io/t/how-to-use-different-bundles-with-different-projects/1096).
+:::
+
+### How settings are organized?
+
+Each addon version includes different settings variations:
+
+- **Default Values**: These values are built into the addon version.
```
[the remark this answers is not in the document]
Comment (MustafaJafar) [reply website/docs/admin_server_bundles_and_addons.md]: https://github.com/ynput/ayon-documentation/pull/298/commits/c4e6ebf1b1da8bd66b1e42d5ffaec4207c3a35a7

File: website/docs/admin_server_bundles_and_addons.md [hunk 18]
```diff
@@ -0,0 +1,292 @@
...
+- You can set **Production** and **Staging** to the same bundle to experiment with different settings, or to two different bundles to test different addon versions.
+- A bundle marked as **Dev** cannot be changed to **Production** or **Staging**. You must `unset Dev` before changing its status.
+- Dev bundles are exclusive to one developer, so each developer should have their own dev bundle.
+- The AYON launcher supports selecting the pipeline mode—**Production**, **Staging**, or **Dev**—which automatically picks the bundle associated with the selected mode.
+
+:::tip Override Pipeline Mode
+The AYON launcher lets you override the default behavior, allowing you to specify a bundle and pipeline mode. For more details, visit [How to use different bundles with different projects? | Ynput Forums](https://community.ynput.io/t/how-to-use-different-bundles-with-different-projects/1096).
+:::
+
+### How settings are organized?
```
[the remark this answers is not in the document]
Comment (MustafaJafar) [reply website/docs/admin_server_bundles_and_addons.md]: https://github.com/ynput/ayon-documentation/pull/298/commits/c4e6ebf1b1da8bd66b1e42d5ffaec4207c3a35a7

File: website/docs/admin_server_bundles_and_addons.md (hunk 16 above)
Comment (MustafaJafar) [reply website/docs/admin_server_bundles_and_addons.md]: https://github.com/ynput/ayon-documentation/pull/298/commits/c4e6ebf1b1da8bd66b1e42d5ffaec4207c3a35a7

File: website/docs/admin_server_bundles_and_addons.md (hunk 15 above)
Comment (MustafaJafar) [reply website/docs/admin_server_bundles_and_addons.md:106]: That's fair! 
I'm not sure what are the limitations. does it limit the uploading custom bundles only?

Comment (BigRoy) [reply website/docs/admin_server_bundles_and_addons.md:106]: As far as I know custom addon - and you can get addons only through marketplace - but it may only be the trial account. Not sure.

File: website/docs/admin_server_bundles_and_addons.md (hunk 13 above)
Comment (Innders) [reply website/docs/admin_server_bundles_and_addons.md:102]: Upload Addons, Launchers and Dependency Packages.

File: website/docs/admin_server_bundles_and_addons.md (hunk 14 above)
Comment (MustafaJafar) [reply website/docs/admin_server_bundles_and_addons.md]: https://github.com/ynput/ayon-documentation/pull/298/commits/7f24457383569e711b4cb6cacd6f622509cb02e8

File: website/docs/admin_server_bundles_and_addons.md (hunk 13 above)
Comment (MustafaJafar) [reply website/docs/admin_server_bundles_and_addons.md]: https://github.com/ynput/ayon-documentation/pull/298/commits/7f24457383569e711b4cb6cacd6f622509cb02e8

File: website/docs/admin_server_bundles_and_addons.md (hunk 15 above)
Comment (MustafaJafar) [reply website/docs/admin_server_bundles_and_addons.md]: https://github.com/ynput/ayon-documentation/pull/298/commits/d46882657b013e0b1752ce12121ed627d4268fdd

**Comment by MustafaJafar** [comment]
I've added it. also, note that it mentions a not yet published community guide. but it should be published today or tomorrow.

**Comment by MustafaJafar** [comment]
For information, the following [How to Test Addons in Active Development on GitHub](https://community.ynput.io/t/how-to-test-addons-in-active-development-on-github/2203) has some links that should be available once this PR is merged.

**Comment by MustafaJafar** [comment]
I've just noticed that it misses information about dependency packages and when and how to update them🙄.

**Comment by MustafaJafar** [comment]
Ok, I've added some information about dependency packages.

File: website/docs/admin_server_bundles_and_addons.md [hunk 19]
```diff
@@ -0,0 +1,322 @@
...
+    )}
+>
+    {addons.slice(0, 6).map((addon, i) => (
+        <AddonCard key={addon.title} addon={addon} />
+    ))}
+</ul>
+
+### Addons Tab
+
+The Addons Tab in the AYON server is used to manage your addon packages.
```
Comment (LiborBatek) [inline website/docs/admin_server_bundles_and_addons.md:35]:
```suggestion
The "Addons" tab in the AYON server is used to manage your addon packages. Navigate to the "Studio Settings" (S+S hotkey combo) and switch to "Addons" pane at the very top left to display all installed addons their status and available versions.
```

**Review by LiborBatek — CHANGES_REQUESTED** [review]
Apart from small addition of info how to navigate to the ```Addons``` tab I also propose to swtich one picture with marked up color codes on the addons settings and what they mark...

Improving the informational value imho...

here is the orig and proposal for improvement:

settings_and_overrides_colorcode-630866b7e0ff59627794b6bc52d8b6fd

and the new one?
settings_and_overrides_colorcode_rev1

**Comment by philippe-ynput** [comment]
How about that one @MustafaJafar ? :-)

**Comment by MustafaJafar** [comment]
> How about that one @MustafaJafar ? :-)

This is one good to go.
Feel free to approve it and then I can merge. 
Please, Let me know if you have further comments ^^

File: website/docs/admin_server_bundles_and_addons.md [hunk 20]
```diff
@@ -0,0 +1,322 @@
...
+
+### Install Addons
+
+You can install addons to your AYON server in a few ways:
+1. **Using Addon Market:** Search for addon releases and choose which one to install. Check out the [Addon Market](admin_server_market.md#market-addons) for more details.
+2. **Upload Addon `.zip` Packages:** Use the **Upload Addons** option on the [Addons](#addons-tab) or [Bundles](#bundles-tab) tab. This is usually for creating `.zip` packages for addons that aren't released yet, GitHub pull requests, or packages shared within the community. For more information, see [How to Test Addons in Active Development on GitHub | Yn Forums](https://community.ynput.io/t/how-to-test-addons-in-active-development-on-github/2203).
+
+:::info Add Installed Addons to Your Pipeline
+
+To add addons to your pipeline, [create a bundle](#create-a-bundle) and include or update the desired addons in it. Assign a pipeline mode to the bundle. More details about bundles and how to create them are in the next section.
```
Comment (BigRoy) [inline website/docs/admin_server_bundles_and_addons.md:77]:
```suggestion
To add addons to your pipeline, [create a bundle](#create-a-bundle) and include or update the desired addons in it. Assign a [pipeline mode](#pipeline-modes) to the bundle. More details about bundles and how to create them are in the next section.
```

File: website/docs/admin_server_bundles_and_addons.md [hunk 21]
```diff
@@ -0,0 +1,322 @@
...
+
+For more details on customizing your pipeline, please refer to:
+- [Dev Docs](dev_introduction.md)
+- [`addon-dev` tag | Ynput Forums](https://community.ynput.io/tag/addon-dev)
+
+:::
+
+:::caution Addons Upload Restriction
+
+[Ynput Cloud](https://ynput.cloud/) instances restrict custom addon uploads by default.
```
Comment (BigRoy) [inline website/docs/admin_server_bundles_and_addons.md:127]: Is this true for cloud instances, or only for trial instances? @mkolar

File: website/docs/admin_server_bundles_and_addons.md [hunk 22]
```diff
@@ -0,0 +1,322 @@
...
+:::
+
+### Dependency Packages
+
+Dependency packages are essential in bundles as they include the necessary Python dependencies for the addons' client code to function properly.
+
+Once a bundle is created, you can set or update the dependency package for each operating system. Clicking the `+` button shows a dropdown menu where you can select a dependency package.
+
+:::tip Get Dependency Packages
+When [updating the pipeline to the latest release](admin_server_updating_pipeline.md), you receive a bundle and a compatible dependency package. In some cases, when building a custom bundle, you may need to build a dependency package for it. For this, use the [ayon-dependencies-tool](https://github.com/ynput/ayon-dependencies-tool). For more information, check out this community [guide: What are dependency packages and how to update them? | Yn Forums](https://community.ynput.io/t/what-are-dependency-package-and-how-to-update-them/1991).
```
Comment (BigRoy) [inline website/docs/admin_server_bundles_and_addons.md:175]:
```suggestion
When [updating the pipeline to the latest release](admin_server_updating_pipeline.md), you receive a bundle and a compatible dependency package. In some cases, when building a custom bundle, you may need to build a dependency package for it. For this, use the [ayon-dependencies-tool](https://github.com/ynput/ayon-dependencies-tool). For more information, check out this community [guide: What are dependency packages and how to update them? | Ynput Forums](https://community.ynput.io/t/what-are-dependency-package-and-how-to-update-them/1991).
```

File: website/docs/admin_server_bundles_and_addons.md [hunk 23]
```diff
@@ -0,0 +1,322 @@
...
+:::tip
+Developers can upload or remove addons using the AYON Python API.  
+For example, see [AYON Python API First Steps: Upload Addon | Ynput Forums](https://community.ynput.io/t/ayon-python-api-first-steps/1278#upload-addon-7).
+:::
+
+### Install Addons
+
+You can install addons to your AYON server in a few ways:
+1. **Using Addon Market:** Search for addon releases and choose which one to install. Check out the [Addon Market](admin_server_market.md#market-addons) for more details.
+2. **Upload Addon `.zip` Packages:** Use the **Upload Addons** option on the [Addons](#addons-tab) or [Bundles](#bundles-tab) tab. This is usually for creating `.zip` packages for addons that aren't released yet, GitHub pull requests, or packages shared within the community. For more information, see [How to Test Addons in Active Development on GitHub | Yn Forums](https://community.ynput.io/t/how-to-test-addons-in-active-development-on-github/2203).
```
Comment (BigRoy) [inline website/docs/admin_server_bundles_and_addons.md:73]:
```suggestion
2. **Upload Addon `.zip` Packages:** Use the **Upload Addons** option on the [Addons](#addons-tab) or [Bundles](#bundles-tab) tab. This is usually for creating `.zip` packages for addons that aren't released yet, GitHub pull requests, or packages shared within the community. For more information, see [How to Test Addons in Active Development on GitHub | Ynput Forums](https://community.ynput.io/t/how-to-test-addons-in-active-development-on-github/2203).
```
What's the reason for abbreviating Ynput to Yn?

**Review by BigRoy — APPROVED** [review]
Looking good! Had some minor notes - but I think once addressed this is good to go.

File: website/docs/admin_server_bundles_and_addons.md (hunk 21 above)
Comment (MustafaJafar) [reply website/docs/admin_server_bundles_and_addons.md:127]: I think any Ayon subscription even if it were on premise.

File: website/docs/admin_server_bundles_and_addons.md (hunk 23 above)
Comment (MustafaJafar) [reply website/docs/admin_server_bundles_and_addons.md:73]: I thought it's a valid shortcut since we are saying `Yn Summit` instead of `Ynput Summit`.

File: website/docs/admin_server_bundles_and_addons.md [hunk 24]
```diff
@@ -0,0 +1,322 @@
...
+- **Development variations**: They are added on demand when the addon is included in a dev bundle. The variation name matches the name of the dev bundle that includes the addon.
+  
+:::info
+The following example assumes a dev bundle named `Develop`.
+:::
+
+![](assets/server/admin/server_and_addons/addon_settings.png)<br/>
+*Example addon with three variations: Production, Staging, and Develop.*
+
+**It's important to remember that bundles only group the addons for use in the AYON pipeline, and any settings values are saved within the addons themselves. This explains why the same addon retains its production and staging values when added to other bundles marked as production or staging.**
```
[the remark this answers is not in the document]
Comment (MustafaJafar) [reply website/docs/admin_server_bundles_and_addons.md:224]: I've to admit it's hard to explain this topic without going further into dev details since this is an admin doc not dev doc.

File: website/docs/admin_server_bundles_and_addons.md (hunk 22 above)
Comment (MustafaJafar) [reply website/docs/admin_server_bundles_and_addons.md]: https://github.com/ynput/ayon-documentation/pull/298/commits/5a10b44dca21adbd20ca6a2312ae80b66927b5b1

File: website/docs/admin_server_bundles_and_addons.md (hunk 23 above)
Comment (MustafaJafar) [reply website/docs/admin_server_bundles_and_addons.md]: https://github.com/ynput/ayon-documentation/pull/298/commits/5a10b44dca21adbd20ca6a2312ae80b66927b5b1

File: website/docs/admin_server_bundles_and_addons.md (hunk 20 above)
Comment (MustafaJafar) [reply website/docs/admin_server_bundles_and_addons.md]: https://github.com/ynput/ayon-documentation/pull/298/commits/5a10b44dca21adbd20ca6a2312ae80b66927b5b1

File: website/docs/admin_server_bundles_and_addons.md (hunk 21 above)
Comment (MustafaJafar) [reply website/docs/admin_server_bundles_and_addons.md]: https://github.com/ynput/ayon-documentation/pull/298/commits/16c62d92c0746a52baddc2f6a9b5e8eba94ec92c

**Review by LiborBatek — DISMISSED** [review]
I have got 2 things to discuss:

1) the image used for a ```Warning``` label is huge and enlarge the widget to extreme...Im for resize it to smaller size and/or put it outside of the warning widget to keep its formatting sensibly large (I prefer it)


image



2) Section for ```Settings and Overrides``` lacks some intro paragraph what it is about. So maybe putting something like:

```
Each AYON addon has its own settings that can be adjusted to suit user preferences on multiple levels. When the default settings are changed, an "override" is created. You can create overrides at either the "Studio" or "Project" level.

A "Studio override" affects all projects and their add-on settings. In contrast, a "Project override" only applies to the selected project and doesn't impact others. You can also use a "Project override" to adjust a "Studio override" without any problems, allowing you to tweak the add-on settings to your desired state.

::note
It's recommended to perform overrides at the studio level first so they apply to all projects. Use project-level overrides only when necessary, as this makes managing addons settings easier later on!
::
```

**Comment by BigRoy** [comment]
> 1. the image used for a `Warning` label is huge and enlarge the widget to extreme...Im for resize it to smaller size and/or put it outside of the warning widget to keep its formatting sensibly large (I prefer it)

I'd say, remove the image.

**Comment by LiborBatek** [comment]
> I'd say, remove the image.

Yep, from the info/warning widget at least...

@MustafaJafar  could you use this formatting for the hotkey addition for accesing the Studio Settings via S+S key too?

as for example here (as I was not aware of syntax when suggested that change)

image

**Comment by MustafaJafar** [comment]
> @MustafaJafar could you use this formatting for the hotkey addition for accesing the Studio Settings via S+S key too?
> 
> as for example here (as I was not aware of syntax when suggested that change)
> 
> image


Do you mean to: 
- ✅ do this:
image
- ❌Not This:
image

Also, does doing it for the `Addon Settings Categories` section only is sufficient?

File: website/docs/assets/server/admin/server_and_addons/start_a_conversation.png
[the remark this answers is not in the document]
Comment (MustafaJafar) [reply website/docs/assets/server/admin/server_and_addons/start_a_conversation.png:1]: The image was removed here https://github.com/ynput/ayon-documentation/pull/298/commits/9e0a22aa15383a93f4fef186f36d106d220ab297

File: website/docs/admin_server_bundles_and_addons.md [hunk 25]
```diff
@@ -0,0 +1,332 @@
...
+:::info Add Installed Addons to Your Pipeline
+
+To add addons to your pipeline, [create a bundle](#create-a-bundle) and include or update the desired addons in it. Assign a [pipeline mode](#pipeline-modes) to the bundle. More details about bundles and how to create them are in the next section.
+:::
+
+## Bundles
+
+### What is a Bundle?
+
+A bundle is a collection of AYON products used in the AYON pipeline. It includes:
```
Comment (dee-ynput) [inline website/docs/admin_server_bundles_and_addons.md:84]: "Product" means something else in AYON :grimacing:

Comment (MustafaJafar) [reply website/docs/admin_server_bundles_and_addons.md:84]: Great point!

Comment (MustafaJafar) [reply website/docs/admin_server_bundles_and_addons.md:84]: what about: 
```suggestion
A bundle is a collection of AYON Pipeline components used in the AYON pipeline. It includes:
```

Comment (Innders) [reply website/docs/admin_server_bundles_and_addons.md:84]: I would suggest that: A bundle is a collection of AYON integrations (Addons) used in your pipeline.

**Comment by philippe-ynput** [comment]
It looks like we should have a glossary page somewhere to make sure we use the correct terminology everywhere ?

**Comment by MustafaJafar** [comment]
> It looks like we should have a glossary page somewhere to make sure we use the correct terminology everywhere ?

We have the [Key concepts glossary](https://ayon.ynput.io/docs/artist_concepts)

File: website/docs/admin_server_bundles_and_addons.md (hunk 25 above)
Comment (MustafaJafar) [reply website/docs/admin_server_bundles_and_addons.md]: https://github.com/ynput/ayon-documentation/pull/298/commits/d9345ed55305dfa13f9e7c3b859985244fc408ef

File: website/docs/admin_server_bundles_and_addons.md (hunk 24 above)
[the remark this answers is not in the document]
Comment (MustafaJafar) [reply website/docs/admin_server_bundles_and_addons.md:226]: @martastain what do you think

[the remark this answers is not in the document]
Comment (martastain) [reply website/docs/admin_server_bundles_and_addons.md:226]: I agree with @BigRoy here. Maybe something like this?
```
Addon settings are stored for each addon version and variant (not per bundle), that's why the same addon retains its production and staging values when added to other bundles marked as production or staging
```

File: website/docs/admin_server_bundles_and_addons.md (hunk 3 above)
Comment (MustafaJafar) [reply website/docs/admin_server_bundles_and_addons.md]: https://github.com/ynput/ayon-documentation/pull/298/commits/462ad1a26e53efe037e6190da047f8232f77256e
